docs/configuration.md

```diff
@@ -94,3 +94,10 @@ An identifier for the assumed role session
 The relative path that is used to fetch credentials inside and ECS instance.
 See [IAM Roles for Tasks](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-iam-roles.html) for more information.
 
+### pathStyleEndpoint
```

Comment:
Should we name this to `s3PathStyleEndpoint`?

Reply:
this comment conviced me not https://github.com/async-aws/aws/pull/630#discussion_r427631052

At least, I can create a upper section `## S3 Specific configuration`